challenges/backend/src/app/services/Config/classes/Config.ts

```diff
@@ -0,0 +1,24 @@
+import { injectable } from "inversify";
+import "reflect-metadata";
+import { IConfig } from "../interface/IConfig";
+
+
+@injectable()
+export class Config implements IConfig {
```

Comment:
No need to create a class/interface set only to map the environment variables.
It would exist if there was a need to validate certain variables or create specific functions, then you could create an enum and work them within the classes.

As a good practice, in our specific case, it would be interesting to create .env files (e.g. .env.dev) and not forget to inform them in the .gitignore file.

Reply:
Hi @johnoliv 

Thank you for the review it was helpful.
For using lint I prefer using [gts](https://github.com/google/gts) and also I've got a linter extension but prefer to not modify the dependencies. 

Thank you